src/pages/404.astro

```diff
@@ -3,31 +3,33 @@ import Main from '../layouts/MainCentered.astro'
 ---
 
 <Main>
-<section>
-<h1> <span class="pink">4</span><span class="white">0</span><span class="blue">4</span></h1>
-<p> Oh No The page you where looking for was not found</p>
-</section>
+	<section>
+		<h1>
+			<span class="pink">4</span>
+			<span class="white">0</span>
+			<span class="blue">4</span>
+		</h1>
+		<p> Oh no! The page you where looking for was not found.</p>
+	</section>
 </Main>
 
 <style>
-    h1 {
-        font-size: 5em;
-    }
-    p {
-        font-size: 1.5em;
-    }
-    .pink {
-        color: var(--secondary);
-    }
-    .white {
-        color: var(--white);
-    }
-    .blue {
-        color: var(--tertiary);
-    }
-    section {
-        font-family: var(--mainfont);
-        padding-top: 150px;
-        text-align: center;
-    }
+	section {
+		padding-top: 60px;
+		text-align: center;
+	}
+
+	h1 {
+		font-size: 8rem;
+		word-spacing: -0.4ch;
```

Comment:
Since the `<h1>` has 3 `<span>`s inside, and each `<span>` is on a newline, it's rendered as having a space in-between (like: `<h1> 4 0 4 </h1>`). This is actually great because it means screenreaders should read it as "four-zero-four", rather than "four-hundred and four". However, the spaces make the title look disjointed. `word-spacing` adjusts the space between words, which I use here to tighten back to looking like one word.